class-job.php

```diff
@@ -51,6 +51,8 @@ public function save() {
 			$data['interval'] = $this->interval;
 		}
 
+		wp_cache_delete( "jobs:{$this->site}", 'cavalcade-jobs' );
```

Comment:
The issue is if Cavalcade uses the persistent cache, but the runner doesn't, then the cache will fall out-of-sync with the database. For example, after a recurring job is completed, the job is rescheduled for the next interval. To do this, the runner updates the database with the new time, but the cache won't be updated, so Cavalcade wouldn't see the new time.

This can be problematic in particular with `wp_next_scheduled()`, as it would reflect the wrong time (or potentially, even show a job as scheduled when it isn't).